### ISSUE TITLE
Tap Pagination gets automatically enabled if Immersive Mode is enabled when opening a book 

### ISSUE THREAD

**Issue by morning-paradigm** [Kareadita/Kavita#1997]
**Describe the bug**
I have `Tap Pagination` disabled and `Immersive Mode` enabled in user preferences. Despite that, `Tap Pagination` gets automatically enabled every time I open a book (epub if that matters).
If `Immersive Mode` is disabled in user preferences, the `Tap Pagination` setting works as expected.

**To Reproduce**
1. Go to user preferences
2. Disable `Tap Pagination` and enable `Immersive Mode` (order doesn't matter)
3. Open any book
4. See `Tap Pagination` is enabled

**Expected behavior**
The two settings should be independent.

**Desktop (please complete the following information):**
 - OS: Docker
 - Browser: Vivaldi, Firefox
 - Version 0.7.2.0 (yes, I did update my container as with the repush)

**Comment by majora2007** [comment]
This is per design. In immersive mode, you do not have a menu bar always present, so you need a way to move to the next page (if in column mode) or chapter (scroll), hence why tap to pagination is automatically enabled.

**Comment by morning-paradigm** [comment]
That seems like something that applies more to touchscreen devices, in which case I agree (although I'd personally prefer swiping if there was such an option).

I mostly read on my PC, where I use arrow keys, so having tap pagination automatically enabled only causes random misclicking when deselecting text. Clicking inside the text column unhides the menu bar, which requires an additional click to hide it again. That's why clicking the screen edges with tap pagination disabled is the best option for deselecting text without it toggling the menu bar or changing pages.

**Comment by FireFtw** [comment]
I use the same setup as @morning-paradigm. Regardless of arrow keys, on desktop clicking the page will toggle the menu bar off and on with immersive mode enabled. I don't understand why you should have to view a permenant menu bar when it's so easily toggled on and off.

You can already do this setup by turning paginate off after enabling immersive mode, it's just impossible to have your user preferences set up to do it by default.

**Comment by Hobogrammer** [comment]
I agree that these two options should be untied from each other. If user confusion is an issue a tooltip could be added to educate the user. Or the feature could be renamed to "auto hide menu" or something to make clearer.
